src/main/java/io/lumigo/core/SpansContainer.java

```diff
@@ -63,6 +63,9 @@ public void init(Map<String, String> env, Reporter reporter, Context context, Ob
         this.clear();
         this.reporter = reporter;
         awsTracerId = env.get(AMZN_TRACE_ID);
+        if (awsTracerId == null) {
+            awsTracerId = System.getProperty("com.amazonaws.xray.traceHeader");
```

Comment:
This is a mandatory change - I contacted support and this is the new approach since java17